samples/kotlin-rulesets-creating/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+    `java`
```

Comment:
I think you can remove `java` plugin.

Reply:
This project is java library project, so if I remove this, gradle cannot resolve any references.
After removing this:
```
CONFIGURE FAILED in 1s
Script compilation errors:

  Line 11:     compileOnly(kotlin("stdlib"))
               ^ Unresolved reference: compileOnly

  Line 12:     compileOnly("com.github.shyiko.ktlint:ktlint-core:${SamplesVersions.ktlintCore}")
               ^ Unresolved reference: compileOnly
```
refs:
- https://github.com/shyiko/ktlint/blob/master/ktlint-ruleset-template/build.gradle#L14
- https://github.com/shyiko/ktlint/blob/master/ktlint-ruleset-template/build.gradle#L50

Reply:
Just change to:
```kotlin
dependencies {
    "compileOnly"(kotlin("stdlib"))
    "compileOnly"("com.github.shyiko.ktlint:ktlint-core:${SamplesVersions.ktlintCore}")
}
```

Then build will succeed.

Reply:
Fixed 70e5ee8

Reply:
Fixed 94b2714